posts/future-of-TQ.md

```diff
@@ -0,0 +1,53 @@
+---
+title: The Future of Tomatenquark
+description: Outline for Tomatenquark going forward as the project transitions to New Management
+date: 11-12-20 (mm-dd-yy)
+sidebar: false
+---
+ 
+# The Future of Tomatenquark 
+
+Howdy Tomatoes🍅,
+
+First off, I want to assure that Tomatenquark is still very much alive.
+
+![tq-banner]
+
+### New Management 
+----
+
+
+The team is working on transitioning to a new management structure in the wake of Fohlen's recent [announcement](https://tomatenquark.org/posts/experiment-ended.html). We are working to make sure this project is more resilient than any one developer.
+
+Special thanks to Fohlen for everything he's done and all the support he's continuing to provide in the transition process, as well as to everyone else who's work has gotten us to where we are today. 
+
+For now development will focus primarily on maintaining parody with Sauerbraten as it prepares for it's new release, and as we bring new teamembers up to speed. 
+
+### Community First, Always
+---
+
+This is and will remain a community first project, and we're going to need your help going forward. The dream of a fully open-source, modern, community-driven Cube engine game is alive and well.
+
+ Stay posted on this [blog](https://tomatenquark.org/POSTS/) and our [discord](https://discord.gg/47rkQar) for active community developments. 
+
+We also have a brand new [Mastadon](https://mastodon.technology/web/accounts/353138) account, so feel free to send us a toot at [tomatenquark@mastadon.technology](https://mastodon.technology/web/accounts/353138) !
+
+### Annoncing 🎮 **Friday Frag Fests** 🎮
+
+----
+
+![fff-banner]
+
+Finally, I would like to announce starting on **Friday, November 13th** at **8pm EST**, the first weekly **Friday Frag Fest**.
+
+ All are welcome to join us for some good old-fashioned fps fun. Allowing the community a time to congregate should help with some of our "population issues" and bring the fans of this game together!
+
+![weekly-ff]
+
+🍅Hope to see you all there🍅
+
+
+[tq-banner]: https://steamcdn-a.akamaihd.net/steam/apps/1274540/header.jpg?t=1588988887
+[tq-logo]: https://raw.githubusercontent.com/tomatenquark/tomatenquark.github.io/site/images/tomatenquark.png
+[fff-banner]: https://i.imgur.com/6kQFvJ5.png
+[weekly-ff]: https://i.imgur.com/b1BECvg.png
```

Comment:
You could remove these lines and replace their references with links to the ʻimages` folder instead of an external source, something like:

```
![tq-logo](../images/tomatenquark.png "tomatenquark logo")
![fff-banner](../images/posts/friday-frag-fest.png "friday frag fest")
![weekly-ff](../images/weeklyfragfests_friday13_banner.png  "weekly frag fast event")
```

Reply:
Changes made. And fair point about the author thing, I was following the model of fohlen's old posts without it